src/components/graph/ExerciseGraphNode.ts

```diff
@@ -0,0 +1,23 @@
+import { uuid } from './graph-types'
+
+export class ExerciseGraphNode {
```

Comment:
I was wondering if this could instead be an interface instead of a class, as there doesn't seem to be any behavior attached to it.

Reply:
I don't know, maybe? I don't know what the difference means, being so new to typescript.

Reply:
It can't be an interface, because it has behaviour: it has the `constructor`!

Reply:
Well yes, but the constructor doesn't actually seem to do much other than copying fields :)